tests/python/unittest/test_pass_verify_gpu_code.py

```diff
@@ -0,0 +1,161 @@
+"""Test gpu code verifier"""
+import tvm
+
+global valid
```

Comment:
always avoid use of global variable, to carry state, you can use a closure to capture a list